snuba/web/views.py

```diff
@@ -384,8 +381,6 @@ def ensure_tables_migrated(force: bool = False) -> None:
     def write(*, dataset: Dataset):
         from snuba.processor import InsertBatch
 
-        ensure_tables_migrated()
```

Comment:
Do migrations get executed if you just download snuba from source ad run the tests ?

Reply:
Yeah. Right now the migrations for tests are run per dataset when you inherit from BaseTest. We should probably change this in future so we run the same migrate script, but there shouldn't be an impact regarding removing these lines.